runtime/runtime-params-estimator/build.rs

```diff
@@ -0,0 +1,100 @@
+use std::io::Read;
+use std::path::Path;
+use std::path::PathBuf;
+use std::process::Command;
+use std::{env, fs, io, process};
+
+fn main() {
+    if let Err(err) = try_main() {
+        eprintln!("{}", err);
+        process::exit(1);
+    }
+}
+
+fn try_main() -> io::Result<()> {
+    let mut cmd = Command::new("rustup");
+    cmd.args(&["target", "add", "wasm32-unknown-unknown"]);
+    check_status(cmd)?;
+
+    let res = Path::new("./test-contract/res");
+
+    fs::create_dir_all(res)?;
+    build_contract("small", &res.join("smallest_contract.wasm"))?;
+
+    for mode in ["stable", "nightly"] {
+        let make_payload = {
+            let features = if mode == "nightly" { "nightly_protocol_features" } else { "" };
+            let baseline = res.join(format!("{}_baseline_contract.wasm", mode));
+            build_contract(features, &baseline)?;
+            let size = std::fs::metadata(&baseline)?.len();
+
+            move |desired_size: u64| -> io::Result<()> {
+                let payload_size = desired_size.saturating_sub(size);
+                let urandom = fs::File::open("/dev/urandom")?;
```

Comment:
Will only work on Linux. Also makes tests somewhat non-deterministic.

Reply:
Yeah, I've just directly ported whatever was originally in bash: 
https://github.com/near/nearcore/blob/8f90b9267e6b672bab4ee6b483369e66c281b349/runtime/runtime-params-estimator/test-contract/build.sh#L42

And, to be fair, another big (but bad) motivation for me was to use `io::copy + .take` combo, which is quite nice, but indeed completely unnecessary here :D Will replace with in-process generation while I am at it!

Reply:
fixed in https://github.com/near/nearcore/pull/4858/commits/9877c249cc1d9e49ad7ce4c23ab0839ed1f073d4, PTAL